### ISSUE TITLE
Output format control

### ISSUE THREAD

**Issue by JamesStallings** [mpolinowski/nodered-cryptocurrency-dashboard#1]
To call this an issue, I think, substantially overstates the case. 

Basically, I've duplicated the flow, injected (other) ticker symbols, and it all works quite smashingly, and thank you! :100: 

My 'issue' is that I've made four charts now, covering Ethereum, Bitcoin, Cardano and Polygon. I'm not certain which is correct, so to speak, but in the first two charts, the text of the bid/ask amounts appears above the charts, and in the other two charts, they're on the bottom.

Is this something I've done while stumbling around changing the flows to reflect their new datapoints in their outputs? or something beyond our control to manipulate?

Cheers
James

**Comment by mpolinowski** [comment]
Hi James,

I just tried running the dashboard - but it seems the API is blocked from where I am currently located. My flows are running into timeouts. So I just injected the old JSON example from my old README file a couple of times to get the graph going:

![2021-12-14_21-01](https://user-images.githubusercontent.com/18694428/146003201-81ea2edf-3133-406f-9cbe-1d7efb560548.png)


Oh wait - now I get what you mean. The order in which children are displayed inside their parent group can be adjusted in the Dashboard/Layout menu:

![2021-12-14_21-09](https://user-images.githubusercontent.com/18694428/146004394-25e645da-dd7b-42fc-b320-9fca616134bd.png)

Drag them in the right place and hit deploy. If they are already in the correct order, mess them up, deploy, reorder, deploy to resync them to the correct state.
